dev/cnf/oss_dependencies.maven

```diff
@@ -430,8 +430,8 @@ org.eclipse.persistence:org.eclipse.persistence.core:2.7.10
 org.eclipse.persistence:org.eclipse.persistence.core:3.0.2
 org.eclipse.persistence:org.eclipse.persistence.jpa.jpql:2.7.10
 org.eclipse.persistence:org.eclipse.persistence.jpa.jpql:3.0.2
-org.eclipse.persistence:org.eclipse.persistence.jpa.modelgen.processor:3.0.2
 org.eclipse.persistence:org.eclipse.persistence.jpa.modelgen.processor:2.7.10
+org.eclipse.persistence:org.eclipse.persistence.jpa.modelgen.processor:3.0.2
```

Comment:
Did you mean to check this in?  It doesn't look like anything changed except the order.

Reply:
I believe there is a Gradle script that gets run automatically to alphabetize this file with the expectation that developers are to check in the correct when it happens (similar to the dependabot script).